debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -404,7 +408,8 @@ protected void execute() {
                                 else {
                                     logger.info("\t '{}' is not added among known tables", id);
                                 }
-                                if (filters.tableFilter().test(id)) {
+                                //
```

Comment:
Please remove the comment

Reply:
Sure, will re-check elsewhere for the same